release/release-process.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Release Process
+---
+<-- TODO -->
+# How to Update Version
+
+<-- TODO -->
+# How to Tag
+
+<-- TODO -->
+# Test Process
+
+<-- TODO -->
+# Build Process
+
+
+# Publish to Maven Central
+
+## Account Setup
+Following http://central.sonatype.org/ 
+
+1. Request access to publish to com.twitter by creating account and requesting access in https://issues.sonatype.org
+2. Twitter contact needed to confirm access (see example https://issues.sonatype.org/browse/OSSRH-22297)
+3. Install local GPG/PGP to sign files by following http://central.sonatype.org/pages/requirements.html#sign-files-with-gpgpgp 
+4. Publish public key (example http://central.sonatype.org/pages/working-with-pgp-signatures.html#distributing-your-public-key)
```

Comment:
instead of showing full urls, let's link the text here and elsewhere. So 'example' will be a link to the sonatype uri, instead of showing the uri.